lib/java/ydoc-server/src/main/java/org/enso/ydoc/server/Ydoc.java

```diff
@@ -135,6 +173,7 @@ public void start() throws ExecutionException, InterruptedException, IOException
                   var bindings = ctx.getBindings("js");
                   bindings.putMember("YDOC_HOST", hostname);
                   bindings.putMember("YDOC_PORT", port);
+                  bindings.putMember("YDOC_MESSAGE_CALLBACKS", callbacks);
```

Comment:
- registering non-primitive type as global symbol seems strange
- with `YDOC_HOST` one could post it as environment variable, but 
- one cannot register `callbacks` via environment variable